assets/src/amp-story-editor-blocks.js

```diff
@@ -315,11 +319,15 @@ store.subscribe( () => {
 	}
 } );
 
+registerPlugin( 'amp-featured-image-pre-publish', { render: withPrePublishNotice } );
+
 addFilter( 'blocks.registerBlockType', 'ampStoryEditorBlocks/setBlockParent', setBlockParent );
 addFilter( 'blocks.registerBlockType', 'ampStoryEditorBlocks/addAttributes', addAMPAttributes );
 addFilter( 'editor.BlockEdit', 'ampStoryEditorBlocks/addAnimationControls', withAnimationControls );
 addFilter( 'editor.BlockEdit', 'ampStoryEditorBlocks/addStorySettings', withAmpStorySettings );
 addFilter( 'editor.BlockEdit', 'ampStoryEditorBlocks/addPageNumber', withPageNumber );
+addFilter( 'editor.BlockEdit', 'ampStoryEditorBlocks/addPageNumber', withEditFeaturedImage );
```

Comment:
I think `addPageNumber` here is copy/pasta, right?

Reply:
Thanks, you're right. https://github.com/ampproject/amp-wp/pull/1998/commits/b9cd74b2716c354f42be76b60f77cd63eea32bcb changes the hook name.